README.md

```diff
@@ -1,119 +1,189 @@
+
+## Data
+| Subset                   | Images (full-size)                                                                        | Images <br/>(max side size 300px)                                                       | Metadata                                                           |
+|--------------------------|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|--------------------------------------------------------------------|
+| Danish Fungi 2020        | [LINK [~110GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-train_val.tar.gz) | [LINK [~6.5GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-train_val.tar.gz) | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-metadata.zip) |
+| Danish Fungi 2020 – Mini | [LINK [~12.5GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20M-images.tar.gz)  | ---                                                                                     | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20M-metadata.zip) |
+| Danish Fungi 2024        | ❗Same as for DF20❗                                                                        | ❗Same as for DF20❗                                                                      | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DanishFungi2024.zip)                                                           |
+| Danish Fungi 2024 – Mini | ❗Same as for DF20 – Mini❗                                                                 | ---                                                                                     |    [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DanishFungi2024-Mini.zip)                                                   |
+
+
+To download the dataset files in CMD, use:
+```
+wget --no-check-certificate URL
+```
 
-Available at -> https://sites.google.com/view/danish-fungi-dataset
+In order to support research in fine-grained plant classification and to allow full reproducibility of our results, we share the training scripts and data tools.
+- Checkpoints are available at [Hugging Face Hub Repository](https://huggingface.co/BVRA).
+- Train and Validation logs are available at [Weights & Biases Workspace](https://wandb.ai/zcu_cv/DanishFungi2023).
 
-## Training
 
-1. Download PyTorch NGC Docker Image and RUN docker container
 
+## Installation
+Python 3.10+ is required.
+### Local instalation
+1. Install dependencies
 ```
-docker pull nvcr.io/nvidia/pytorch:21.07-py3
-docker run --gpus all -it --rm -v local_dir:container_dir nvcr.io/nvidia/pytorch:21.07-py3
+pip install -r requirements.txt
 ```
-
-2. Install dependencies inside docker container
-
+2. Login to [Weights & Biases](https://wandb.ai/site) to log results [*optional].
 ```
-pip install pandas seaborn timm albumentation tqdm efficientnet_pytorch pretrainedmodels
+import wandb
```

Comment:
I would specify what is done in cmd, what in python, etc. Or maybe give all instructions for cmd, ie 
`wandb login
`